lib/new_relic/sampler/top_process.ex

```diff
@@ -76,7 +76,7 @@ defmodule NewRelic.Sampler.TopProcess do
   defp parse(:name, pid, []) do
     with {:dictionary, dictionary} <- Process.info(pid, :dictionary),
          {m, f, a} <- Keyword.get(dictionary, :"$initial_call") do
-      "#{inspect(m)}.#{f}/#{a}"
+      "#{inspect(m)}.#{f}/#{inspect(a)}"
```

Comment:
Something else must be going wrong here. The `a` in `$initial_call` is set by `Task` and is the _arity_ - it is always a number.

https://github.com/elixir-lang/elixir/blob/fb729784e5504f499b0179c4f154e73ea2a6f216/lib/elixir/lib/task/supervised.ex#L77-L85

Are you doing something manually with `$initial_call`?

Reply:
Nope not doing anything manually with initial call:
```
    task_stream =
      Task.Supervisor.async_stream_nolink(
        XXXX.TaskSupervisor,
        batches,
        callback_module,
        :execute,
        [refined_context],
        max_concurrency: callback_module.max_concurrency(),
        timeout: :infinity
      )
```